.restyled.yaml

```diff
@@ -4,5 +4,8 @@ restylers:
     include:
       - './*.{js,md}'
       - 'pages/**/*.js'
-      - 'content/docs/**/*.{js,md}'
+      - 'content/**/*.{js,md}'
```

Comment:
do we have js inside content?

Reply:
I don't think so. Removed `.js` extensions there